### ISSUE TITLE
20NG Training Set Reported Results Question

### ISSUE THREAD

**Issue by jharrang** [yao8839836/text_gcn#133]
For the 20NG results reported in [the original paper](https://arxiv.org/pdf/1809.05679v3.pdf) (`0.8634 ± 0.0009`), was the model trained on the full public 20NG `train` set, of size 11314, or were the reported results generated using the code currently in this repo (as of https://github.com/yao8839836/text_gcn/tree/962223652e9bb164ac2d83cd09fc7b8845ce860b), which appears to exclude the validation set from the data used to train the model? (The latter would use a training set of size 10183).

Thanks!

**Comment by yao8839836** [comment]
@jharrang 

The real training set size is 10183.

**Comment by jharrang** [comment]
@yao8839836 - Thanks so much! Did you consider or attempt training a final model on the full `11314` document train set. I'm wondering if that might give a slightly better result due to the increased training set size.

**Comment by jharrang** [comment]
@yao8839836 - I would still be curious as to whether training on the full `11314` document training set was attempted, but this answered my original question, so I'm going to close the issue. Thanks!
